### PR TITLE
Create VPC

### PR DESCRIPTION
**Pull request by SaintClair33**
Pull request for Terraform creation of VPC

### CODE REVIEW

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]
Please run `terraform fmt` - it will automatically make some code style adjustments. After that commit changes to the same branch

File: vpc/main.tf [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+resource "aws_subnet" "public0" {
+  vpc_id     = aws_vpc.main.id
+  cidr_block = "10.0.0.0/24"
+
+   tags = {
+    Name = "public0"
+  }
+
+}
```
Comment (vladimirmukhin) [inline vpc/main.tf:17]:
```suggestion
  }
}
```

File: vpc/main.tf [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+resource "aws_route_table_association" "private0" {
+  subnet_id      = aws_subnet.private0.id
+  route_table_id = aws_route_table.private0.id
+}
+
+resource "aws_route_table_association" "private1" {
+  subnet_id      = aws_subnet.private1.id
+  route_table_id = aws_route_table.private1.id
+}
```
Comment (vladimirmukhin) [inline vpc/main.tf:145]: Please end each file with a newline

File: vpc/providers.tf [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = "us-east-1"
+}
+
```
Comment (vladimirmukhin) [inline vpc/providers.tf:13]: Please have only one newline at the end of a file

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

**Review by SaintClair33 — COMMENTED** [review]
Vlad did I get all of the changes? I thought I submitted all of the changes you recommended, can you confirm?

**Review by vladimirmukhin — APPROVED** [review]

**Pull request merged**